jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -30,30 +30,19 @@ node {
             [
                 $class: 'ParametersDefinitionProperty',
                 parameterDefinitions: [
+                    commonlib.ocpVersionParam('VERSION'),
                     string(
-                        name: 'COMPONENTS',
-                        description: '(REQUIRED) Brew component name',
-                        defaultValue: "logging-fluentd-container",
-                        trim: true,
+                        name: "ASSEMBLY",
```

Comment:
Would this work for 3.11?

Reply:
Yes, just use the default value 'stream'.